.env.sample

```diff
@@ -10,5 +10,9 @@ SITE_URL=
 # The value for this variable is using the same as SITE_URL unless you use an external CDN
 PUBLIC_URL=
 
+# The BROWSER where the development server will be open
+# To not open a browser use 'BROWSER=none' and for the same-tab feature in MacOS use 'BROWSER="google chrome"')
+BROWSER=
```

Comment:
Keep the variable comment out since it's optional.

Reply:
So you want me to have the description in the readme and just the variable definition in the .env file?

Reply:
No, I said to comment it out: `#BROWSER`. The env variable is optional, not mandatory.